modules/virt-configuring-openshift-root-ca-for-providers.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-about-mtv-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-openshift-root-ca-for-providers_{context}"]
+= Configuring the OpenShift root certificate authority for providers
```

Comment:
Not sure if this is a literal name, but probably worth checking if it needs to use an attribute rather than "OpenShift". I'm not familiar with it so not sure.